README.md

```diff
@@ -21,7 +21,7 @@ There are currently 2 parts to the site, the content and the theming/plugins.
 ### Requirements
 
 - [Node](https://nodejs.org/en/download/): 8+
-- [Yarn](https://yarnpkg.com/lang/en/docs/install/#windows-stable): 1.15+
+- [Yarn](https://yarnpkg.com/lang/en/docs/install/): 1.7 or later
```

Comment:
Did you confirm that everything still builds and the dev server runs with yarn 1.7?

Reply:
I have not, no - I was making the change based on your slack comment. I don't have a CentOS / Strap environment to test in.